ducktape/tests/runner.py

```diff
@@ -87,33 +85,43 @@ def run_all_tests(self):
             # Results from this test, as well as logs will be dumped here
             mkdir_p(self.current_test_context.results_dir)
 
+            result_info = {
```

Comment:
Not really a big deal, but why in a dictionary vs using variables for each of these? Being able to use `kwargs` makes passing the final variables to `TestResult` easy, but every other time you set them pretty awkward since you have to do it through a dict.